apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -122,7 +154,56 @@ export default function RootStateDrawer() {
     ],
   )
 
-  const pageLayout: string = savedPageState.layout
+  const handleConversionToIndexPage = useCallback(() => {
+    const newPageState = {
+      ...savedPageState,
+      content: [],
+      childpages: {
+        layout: "boxes",
+        summary: false,
+        thumbnail: false,
```

Comment:
FYI, we shuold use `DEFAULT_CHILDPAGE_BLOCK`

Reply:
Updated to use `DEFAULT_BLOCKS.childpage` since `DEFAULT_CHILDPAGE_BLOCK` is for server-side use